File: 1-easy/problems/2778-sum-of-squares-of-special-elements/solution.ts
function sumOfSquares(nums: any): boolean | number | string | any {
  """
  :type nums: List[int]
  :rtype: int
  """
  result = 0
  for (i in Array.from({length: 1, int((nums}, (_, i) => i).length**0.5)+1)) {
      if ((nums).length%i) {
          continue
      result += nums[i-1]**2
      if ((nums).length//i != i) {
          result += nums[(nums).length//i-1]**2
  return result;
}

export { sumOfSquares };

File: 1-easy/problems/2778-sum-of-squares-of-special-elements/solution.py
from typing import List, Optional, Dict, Set, Tuple

class Solution:
    def sumOfSquares(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        result = 0
        for i in range(1, int(len(nums)**0.5)+1):
            if len(nums)%i:
                continue
            result += nums[i-1]**2
            if len(nums)//i != i:
                result += nums[len(nums)//i-1]**2
        return result
